assets/dev/js/editor/components/template-library/manager.js

```diff
@@ -575,6 +579,18 @@ const TemplateLibraryManager = function() {
 			_.extend( ajaxParams, templateType.ajaxParams );
 		}
 
+		const ajaxParamsSuccess = ajaxParams.success;
+
+		_.extend( ajaxParams, {
+			success: ( ...args ) => {
+				if ( ajaxParamsSuccess ) {
+					ajaxParamsSuccess( ...args );
+				}
+
+				this.triggerQuotaUpdate();
```

Comment:
Why not just put this in line 55?
But also, I have the same question here, maybe we don't have to add `triggerQuotaUpdate()` all over the place. Just on the `my-templates` route.